src/main/java/com/iota/iri/IRI.java

```diff
@@ -361,9 +116,56 @@ private static void shutdownHook() {
                 }
             }, "Shutdown Hook"));
         }
-    }
 
-    public static boolean validateParams(Configuration configuration, String[] args) throws IOException {
-        return IRILauncher.isValidated(configuration, args);
+        private static IotaConfig createConfiguration(String[] args) {
+            IotaConfig iotaConfig = null;
+            String message = "Configuration is created using ";
+            try {
+                boolean testnet = ArrayUtils.contains(args, Config.TESTNET_FLAG);
+                File configFile = chooseConfigFile(args);
+                if (configFile != null) {
+                    iotaConfig = ConfigFactory.createFromFile(configFile, testnet);
+                    message += configFile.getName() + " and command line args";
+                }
+                else {
+                    iotaConfig = ConfigFactory.createIotaConfig(testnet);
+                    message += "command line args only";
+                }
+                JCommander jCommander = iotaConfig.parseConfigFromArgs(args);
+                if (iotaConfig.isHelp()) {
+                    jCommander.usage();
+                    System.exit(0);
```

Comment:
I am a friend of using a wrapper method to exit the application. Instead of calling System.exit in the code it could be a good idea to create a static IRI.exit(int) method instead. If you confirm i will create a PR.